pkgs/unified_analytics/lib/src/event.dart

```diff
@@ -905,6 +905,32 @@ final class Event {
           },
         );
 
+  /// Event that is sent from the Dart MCP server when a tool is invoked.
```

Comment:
is this only for tools? or should we keep this doc general so that it also includes events like resource reads too?

Reply:
Yeah this is intended to be general, will update the comment